web-app/prisma/migrations/20251017100721_add_job_schedule_table/migration.sql

```diff
@@ -0,0 +1,43 @@
+-- Create Enum
```

Comment:
Based on the scheduleType wouldn't it make sense to add a constraint? If scheduleType is Cron the cron value has to be given and oneTime, the oneTimeAtUtc has to be set?